sklearn_pandas/dataframe_mapper.py

```diff
@@ -209,8 +214,9 @@ def fit(self, X, y=None):
 
         """
         self._build()
-
-        for columns, transformers, options in self.built_features:
+        pbar = tqdm(self.built_features, disable=not self.show_progressbar)
+        for columns, transformers, options in pbar:
+            pbar.set_description("[Fit] %s" % columns)
```

Comment:
What is the effect of setting the description in a disabled progress bar? No errors, no warnings?

Reply:
yes..no warning/errors will be raised.